atlasdb-cli/build.gradle

```diff
@@ -11,6 +11,11 @@ dependencies {
   compile project(':atlasdb-impl-shared')
   compile 'io.airlift:airline:0.7'
   compile 'com.google.dagger:dagger:2.0.2'
+  compile 'com.fasterxml.jackson.core:jackson-databind:' + libVersions.jackson
+  compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:' + libVersions.jackson
+  compile 'io.dropwizard:dropwizard-jackson:' + libVersions.dropwizard
```

Comment:
yes:
(1) databind for handling JsonNodes (we can potentially avoid this by making a jackson object that looks like the Dropwizard configuration object that doesn't extend from dropwizard)
(2) yaml for reading the config
(3) dropwizard-jackson for having the service discovery object mapper plugin